manifests/d/dxatlas/CwSkimmer/2.1/dxatlas.CwSkimmer.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: dxatlas.CwSkimmer
+PackageVersion: 2.1
+PackageLocale: en-US
+Publisher: dxatlas
+PackageName: CwSkimmer
+License: none
+ShortDescription: Multi-channel Morse code (CW) decoder and analyzer.
+
+

```

Comment:
Oops, sorry, I have added two new lines in my previous suggestions. Please remove them:

```suggestion

```

Reply:
I do